addons/bus/models/ir_websocket.py

```diff
@@ -23,13 +29,14 @@ def _subscribe(self, data):
         channels = set(self._build_bus_channel_list(data['channels']))
         dispatch.subscribe(channels, data['last'], self.env.registry.db_name, wsrequest.ws)
 
-    def _update_bus_presence(self, inactivity_period):
+    def _update_bus_presence(self, data):
```

Comment:
I'm not sure if I like having a generic data argument like this, maybe keep `inactivity_period` as it is and have one param specific for "data to fetch".